src/priceClient/adapters/across.ts

```diff
@@ -0,0 +1,50 @@
+import axios, { AxiosError } from "axios";
+import get from "lodash.get";
+import { Logger, msToS, PriceFeedAdapter, TokenPrice } from "../priceClient";
+
+type AcrossPrice = { price: number };
+
+export class PriceFeed implements PriceFeedAdapter {
+  public readonly platforms = ["ethereum"];
+
+  constructor(public readonly logger: Logger, public readonly name: string, public readonly host?: string) {
+    // Allow host to be overridden for test or alternative deployments.
+    if (this.host === undefined) this.host = "across.to";
+  }
+
+  async getTokenPrice(address: string, currency: string, platform: string): Promise<TokenPrice> {
```

Comment:
Should be marked override (as with getTokenPrices() below, and repeated in coingecko.ts).

Reply:
Have confirmed that override is used for extending classes, not implementing interfaces...so this is irrelevant.